In [132]:
from bs4 import BeautifulSoup
import requests
import pandas as pd
import sqlite3

**Scrapping info departamentos**

In [144]:
url = 'https://es.wikipedia.org/wiki/Anexo:Departamentos_de_Colombia_por_poblaci%C3%B3n'
pagina = requests.get(url)                           
soup = BeautifulSoup(pagina.content,'html.parser')

In [145]:
tabla_info_dep = soup.find_all('table')

In [146]:
info_dep = tabla_info_dep[0].findChildren('td')

**Conseguimos los links de las banderas**

In [147]:
banderas_dep = info_dep[1::5]                                  #Traemos solo las casillas de la bandera
img_banderas = [i.findChildren('img') for i in banderas_dep]   #De las casillas sacamos el elemento img
src_img_banderas=[i[0].get('src') for i in img_banderas]        #De cada elemento img tomamos el link de la imagen

**Conseguimo los nombre de los departamentos**

In [148]:
nombres_dep =  info_dep[1::5]
text_nombres = [f'{i.text}' for i in nombres_dep]

In [150]:
text_nombres

['\xa0Bogotá, D. C.\n',
 '\xa0Antioquia\n',
 '\xa0Valle del Cauca\n',
 '\xa0Cundinamarca\n',
 '\xa0Atlántico\n',
 '\xa0Santander\n',
 '\xa0Bolívar\n',
 '\xa0Córdoba\n',
 '\xa0Norte de Santander\n',
 '\xa0Nariño\n',
 '\xa0Cauca\n',
 '\xa0Magdalena\n',
 '\xa0Cesar\n',
 '\xa0Tolima\n',
 '\xa0Boyacá\n',
 '\xa0Huila\n',
 '\xa0Meta\n',
 '\xa0Caldas\n',
 '\xa0La Guajira\n',
 '\xa0Risaralda\n',
 '\xa0Sucre\n',
 '\xa0Quindío\n',
 '\xa0Chocó\n',
 '\xa0Casanare\n',
 '\xa0Caquetá\n',
 '\xa0Putumayo\n',
 '\xa0Arauca\n',
 '\xa0Vichada\n',
 '\xa0Guaviare\n',
 '\xa0Amazonas\n',
 '\xa0San Andrés y Providencia\n',
 '\xa0Guainía\n',
 '\xa0Vaupés\n']

In [149]:
poblacion_dep = info_dep[4::5]
text_poblacion = [f'{i.text}' for i in poblacion_dep]
text_poblacion

['7\xa0968\xa0095\n',
 '6\xa0994\xa0792\n',
 '4\xa0622\xa0132\n',
 '3\xa0577\xa0177\n',
 '2\xa0835\xa0509\n',
 '2\xa0340\xa0657\n',
 '2\xa0258\xa0929\n',
 '1\xa0868\xa0166\n',
 '1\xa0658\xa0835\n',
 '1\xa0631\xa0117\n',
 '1\xa0528\xa0076\n',
 '1\xa0476\xa0665\n',
 '1\xa0359\xa0719\n',
 '1\xa0350\xa0060\n',
 '1\xa0267\xa0378\n',
 '1\xa0149\xa0598\n',
 '1\xa0088\xa0749\n',
 '1\xa0046\xa0418\n',
 '1\xa0015\xa0909\n',
 '988\xa0091\n',
 '981\xa0727\n',
 '577\xa0543\n',
 '557\xa0654\n',
 '444\xa0602\n',
 '423\xa0857\n',
 '374\xa0042\n',
 '308\xa0301\n',
 '116\xa0944\n',
 '92\xa0281\n',
 '83\xa0808\n',
 '65\xa0663\n',
 '52\xa0627\n',
 '51\xa0133\n']

In [151]:
info_dep = pd.DataFrame(data={'departamento':text_nombres,'poblacion':text_poblacion, 'bandera':src_img_banderas})
info_dep.sort_values(by=['departamento'], inplace=True)
info_dep

departamento    poblacion  \
29                   Amazonas\n     83 808\n   
1                   Antioquia\n  6 994 792\n   
26                     Arauca\n    308 301\n   
4                   Atlántico\n  2 835 509\n   
0               Bogotá, D. C.\n  7 968 095\n   
6                     Bolívar\n  2 258 929\n   
14                     Boyacá\n  1 267 378\n   
17                     Caldas\n  1 046 418\n   
24                    Caquetá\n    423 857\n   
23                   Casanare\n    444 602\n   
10                      Cauca\n  1 528 076\n   
12                      Cesar\n  1 359 719\n   
22                      Chocó\n    557 654\n   
3                Cundinamarca\n  3 577 177\n   
7                     Córdoba\n  1 868 166\n   
31                    Guainía\n     52 627\n   
28                   Guaviare\n     92 281\n   
15                      Huila\n  1 149 598\n   
18                 La Guajira\n  1 015 909\n   
11                  Magdalena\n  1 476 665\n   
16                       Meta\n  1 088 749\n   
9                      Nariño\n  1 631 117\n   
8          Norte de Santander\n  1 658 835\n   
25                   Putumayo\n    374 042\n   
21                    Quindío\n    577 543\n   
19                  Risaralda\n    988 091\n   
30   San Andrés y Providencia\n     65 663\n   
5                   Santander\n  2 340 657\n   
20                      Sucre\n    981 727\n   
13                     Tolima\n  1 350 060\n   
2             Valle del Cauca\n  4 622 132\n   
32                     Vaupés\n     51 133\n   
27                    Vichada\n    116 944\n   

                                              bandera  
29  //upload.wikimedia.org/wikipedia/commons/thumb...  
1   //upload.wikimedia.org/wikipedia/commons/thumb...  
26  //upload.wikimedia.org/wikipedia/commons/thumb...  
4   //upload.wikimedia.org/wikipedia/commons/thumb...  
0   //upload.wikimedia.org/wikipedia/commons/thumb...  
6   //upload.wikimedia.org/wikipedia/commons/thumb...  
14  //upload.wikimedia.org/wikipedia/commons/thumb...  
17  //upload.wikimedia.org/wikipedia/commons/thumb...  
24  //upload.wikimedia.org/wikipedia/commons/thumb...  
23  //upload.wikimedia.org/wikipedia/commons/thumb...  
10  //upload.wikimedia.org/wikipedia/commons/thumb...  
12  //upload.wikimedia.org/wikipedia/commons/thumb...  
22  //upload.wikimedia.org/wikipedia/commons/thumb...  
3   //upload.wikimedia.org/wikipedia/commons/thumb...  
7   //upload.wikimedia.org/wikipedia/commons/thumb...  
31  //upload.wikimedia.org/wikipedia/commons/thumb...  
28  //upload.wikimedia.org/wikipedia/commons/thumb...  
15  //upload.wikimedia.org/wikipedia/commons/thumb...  
18  //upload.wikimedia.org/wikipedia/commons/thumb...  
11  //upload.wikimedia.org/wikipedia/commons/thumb...  
16  //upload.wikimedia.org/wikipedia/commons/thumb...  
9   //upload.wikimedia.org/wikipedia/commons/thumb...  
8   //upload.wikimedia.org/wikipedia/commons/thumb...  
25  //upload.wikimedia.org/wikipedia/commons/thumb...  
21  //upload.wikimedia.org/wikipedia/commons/thumb...  
19  //upload.wikimedia.org/wikipedia/commons/thumb...  
30  //upload.wikimedia.org/wikipedia/commons/thumb...  
5   //upload.wikimedia.org/wikipedia/commons/thumb...  
20  //upload.wikimedia.org/wikipedia/commons/thumb...  
13  //upload.wikimedia.org/wikipedia/commons/thumb...  
2   //upload.wikimedia.org/wikipedia/commons/thumb...  
32  //upload.wikimedia.org/wikipedia/commons/thumb...  
27  //upload.wikimedia.org/wikipedia/commons/thumb...

In [152]:
info_dep.to_csv(r'info_dep.csv', encoding='utf-8')

<table class="wikitable sortable">
<caption><b>Población de los departamentos de Colombia en 2023</b>
</caption>
<tbody><tr>
<th>N.º
</th>
<th width="200">Departamento
</th>
<th>Población Cabecera
</th>
<th>Población Resto
</th>
<th width="150"><b>Población Total</b>
</th></tr>
<tr align="center">
<td>1
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Bogotá" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/9e/Flag_of_Bogot%C3%A1.svg/20px-Flag_of_Bogot%C3%A1.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/9e/Flag_of_Bogot%C3%A1.svg/30px-Flag_of_Bogot%C3%A1.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/9e/Flag_of_Bogot%C3%A1.svg/40px-Flag_of_Bogot%C3%A1.svg.png 2x" title="Bandera de Bogotá" width="20"> </img></span><a href="/wiki/Bogot%C3%A1" title="Bogotá">Bogotá, D. C.</a>
</td>
<td>7 936 532
</td>
<td>31 536
</td>
<td bgcolor="#00cc33">7 968 095
</td></tr>
<tr align="center">
<td>2
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Antioquia" class="thumbborder" data-file-height="600" data-file-width="900" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/3/33/Flag_of_Antioquia_Department.svg/20px-Flag_of_Antioquia_Department.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/3/33/Flag_of_Antioquia_Department.svg/30px-Flag_of_Antioquia_Department.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/3/33/Flag_of_Antioquia_Department.svg/40px-Flag_of_Antioquia_Department.svg.png 2x" title="Bandera de Antioquia" width="20"> </img></span><a href="/wiki/Antioquia" title="Antioquia">Antioquia</a>
</td>
<td>5 618 672
</td>
<td>1 376 120
</td>
<td bgcolor="#00cc33">6 994 792
</td></tr>
<tr align="center">
<td>3
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Valle del Cauca" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/d/d3/Flag_of_Valle_del_Cauca.svg/20px-Flag_of_Valle_del_Cauca.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/d/d3/Flag_of_Valle_del_Cauca.svg/30px-Flag_of_Valle_del_Cauca.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/d/d3/Flag_of_Valle_del_Cauca.svg/40px-Flag_of_Valle_del_Cauca.svg.png 2x" title="Bandera de Valle del Cauca" width="20"/> </span><a href="/wiki/Valle_del_Cauca" title="Valle del Cauca">Valle del Cauca</a>
</td>
<td>3 964 228
</td>
<td>657 904
</td>
<td bgcolor="#00cc33">4 622 132
</td></tr>
<tr align="center">
<td>4
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Cundinamarca" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Flag_of_Cundinamarca.svg/20px-Flag_of_Cundinamarca.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Flag_of_Cundinamarca.svg/30px-Flag_of_Cundinamarca.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Flag_of_Cundinamarca.svg/40px-Flag_of_Cundinamarca.svg.png 2x" title="Bandera de Cundinamarca" width="20"/> </span><a href="/wiki/Cundinamarca" title="Cundinamarca">Cundinamarca</a>
</td>
<td>2 729 948
</td>
<td>847 229
</td>
<td bgcolor="#00cc33">3 577 177
</td></tr>
<tr align="center">
<td>5
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Atlántico" class="thumbborder" data-file-height="300" data-file-width="500" decoding="async" height="12" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a3/Flag_of_Atl%C3%A1ntico.svg/20px-Flag_of_Atl%C3%A1ntico.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/a3/Flag_of_Atl%C3%A1ntico.svg/30px-Flag_of_Atl%C3%A1ntico.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/a3/Flag_of_Atl%C3%A1ntico.svg/40px-Flag_of_Atl%C3%A1ntico.svg.png 2x" title="Bandera de Atlántico" width="20"/> </span><a href="/wiki/Atl%C3%A1ntico_(Colombia)" title="Atlántico (Colombia)">Atlántico</a>
</td>
<td>2 692 350
</td>
<td>143 159
</td>
<td bgcolor="#00cc33">2 835 509
</td></tr>
<tr align="center">
<td>6
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Santander" class="thumbborder" data-file-height="1800" data-file-width="2700" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/5/53/Flag_of_Santander_Department.svg/20px-Flag_of_Santander_Department.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/5/53/Flag_of_Santander_Department.svg/30px-Flag_of_Santander_Department.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/5/53/Flag_of_Santander_Department.svg/40px-Flag_of_Santander_Department.svg.png 2x" title="Bandera de Santander" width="20"/> </span><a href="/wiki/Santander_(Colombia)" title="Santander (Colombia)">Santander</a>
</td>
<td>1 813 631
</td>
<td>527 026
</td>
<td bgcolor="#00cc33">2 340 657
</td></tr>
<tr align="center">
<td>7
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Bolívar" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Flag_of_Bol%C3%ADvar_%28Colombia%29.svg/20px-Flag_of_Bol%C3%ADvar_%28Colombia%29.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Flag_of_Bol%C3%ADvar_%28Colombia%29.svg/30px-Flag_of_Bol%C3%ADvar_%28Colombia%29.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Flag_of_Bol%C3%ADvar_%28Colombia%29.svg/40px-Flag_of_Bol%C3%ADvar_%28Colombia%29.svg.png 2x" title="Bandera de Bolívar" width="20"/> </span><a href="/wiki/Bol%C3%ADvar_(Colombia)" title="Bolívar (Colombia)">Bolívar</a>
</td>
<td>1 657 859
</td>
<td>601 070
</td>
<td bgcolor="#00cc33">2 258 929
</td></tr>
<tr align="center">
<td>8
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Córdoba" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Flag_of_C%C3%B3rdoba.svg/20px-Flag_of_C%C3%B3rdoba.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Flag_of_C%C3%B3rdoba.svg/30px-Flag_of_C%C3%B3rdoba.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Flag_of_C%C3%B3rdoba.svg/40px-Flag_of_C%C3%B3rdoba.svg.png 2x" title="Bandera de Córdoba" width="20"/> </span><a href="/wiki/C%C3%B3rdoba_(Colombia)" title="Córdoba (Colombia)">Córdoba</a>
</td>
<td>959 723
</td>
<td>908 443
</td>
<td bgcolor="#00cc33">1 868 166
</td></tr>
<tr align="center">
<td>9
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Norte de Santander" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Flag_of_Norte_de_Santander.svg/20px-Flag_of_Norte_de_Santander.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Flag_of_Norte_de_Santander.svg/30px-Flag_of_Norte_de_Santander.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Flag_of_Norte_de_Santander.svg/40px-Flag_of_Norte_de_Santander.svg.png 2x" title="Bandera de Norte de Santander" width="20"/> </span><a href="/wiki/Norte_de_Santander" title="Norte de Santander">Norte de Santander</a>
</td>
<td>1 318 669
</td>
<td>340 166
</td>
<td bgcolor="#00cc33">1 658 835
</td></tr>
<tr align="center">
<td>10
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Nariño" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/20/Flag_of_Nari%C3%B1o.svg/20px-Flag_of_Nari%C3%B1o.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/20/Flag_of_Nari%C3%B1o.svg/30px-Flag_of_Nari%C3%B1o.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/20/Flag_of_Nari%C3%B1o.svg/40px-Flag_of_Nari%C3%B1o.svg.png 2x" title="Bandera de Nariño" width="20"/> </span><a href="/wiki/Nari%C3%B1o_(Colombia)" title="Nariño (Colombia)">Nariño</a>
</td>
<td>716 461
</td>
<td>914 656
</td>
<td bgcolor="#00cc33">1 631 117
</td></tr>
<tr align="center">
<td>11
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Cauca" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Flag_of_Cauca.svg/20px-Flag_of_Cauca.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Flag_of_Cauca.svg/30px-Flag_of_Cauca.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Flag_of_Cauca.svg/40px-Flag_of_Cauca.svg.png 2x" title="Bandera de Cauca" width="20"/> </span><a href="/wiki/Cauca_(Colombia)" title="Cauca (Colombia)">Cauca</a>
</td>
<td>545 597
</td>
<td>982 479
</td>
<td bgcolor="#00cc33">1 528 076
</td></tr>
<tr align="center">
<td>12
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Magdalena" class="thumbborder" data-file-height="216" data-file-width="324" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/1/18/Flag_of_Magdalena.svg/20px-Flag_of_Magdalena.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/1/18/Flag_of_Magdalena.svg/30px-Flag_of_Magdalena.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/1/18/Flag_of_Magdalena.svg/40px-Flag_of_Magdalena.svg.png 2x" title="Bandera de Magdalena" width="20"/> </span><a href="/wiki/Magdalena_(Colombia)" title="Magdalena (Colombia)">Magdalena</a>
</td>
<td>1 008 522
</td>
<td>468 143
</td>
<td bgcolor="#00cc33">1 476 665
</td></tr>
<tr align="center">
<td>13
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Cesar" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/4/43/Flag_of_Cesar.svg/20px-Flag_of_Cesar.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/4/43/Flag_of_Cesar.svg/30px-Flag_of_Cesar.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/4/43/Flag_of_Cesar.svg/40px-Flag_of_Cesar.svg.png 2x" title="Bandera de Cesar" width="20"/> </span><a href="/wiki/Cesar" title="Cesar">Cesar</a>
</td>
<td>1 018 784
</td>
<td>340 935
</td>
<td bgcolor="#00cc33">1 359 719
</td></tr>
<tr align="center">
<td>14
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Tolima" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/b9/Flag_of_Tolima.svg/20px-Flag_of_Tolima.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/b/b9/Flag_of_Tolima.svg/30px-Flag_of_Tolima.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/b9/Flag_of_Tolima.svg/40px-Flag_of_Tolima.svg.png 2x" title="Bandera de Tolima" width="20"/> </span><a href="/wiki/Tolima" title="Tolima">Tolima</a>
</td>
<td>933 687
</td>
<td>416 373
</td>
<td bgcolor="#00cc33">1 350 060
</td></tr>
<tr align="center">
<td>15
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Boyacá" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Flag_of_Boyac%C3%A1_Department.svg/20px-Flag_of_Boyac%C3%A1_Department.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Flag_of_Boyac%C3%A1_Department.svg/30px-Flag_of_Boyac%C3%A1_Department.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Flag_of_Boyac%C3%A1_Department.svg/40px-Flag_of_Boyac%C3%A1_Department.svg.png 2x" title="Bandera de Boyacá" width="20"/> </span><a href="/wiki/Boyac%C3%A1" title="Boyacá">Boyacá</a>
</td>
<td>769 332
</td>
<td>498 046
</td>
<td bgcolor="#00cc33">1 267 378
</td></tr>
<tr align="center">
<td>16
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Huila" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Flag_of_Huila.svg/20px-Flag_of_Huila.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Flag_of_Huila.svg/30px-Flag_of_Huila.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Flag_of_Huila.svg/40px-Flag_of_Huila.svg.png 2x" title="Bandera de Huila" width="20"/> </span><a href="/wiki/Huila" title="Huila">Huila</a>
</td>
<td>690 202
</td>
<td>459 396
</td>
<td bgcolor="#00cc33">1 149 598
</td></tr>
<tr align="center">
<td>17
</td>
<td align="left"><span class="flagicon"><img alt="Bandera del Meta" class="thumbborder" data-file-height="1700" data-file-width="2550" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/7/72/Flag_of_Meta.svg/20px-Flag_of_Meta.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/72/Flag_of_Meta.svg/30px-Flag_of_Meta.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/72/Flag_of_Meta.svg/40px-Flag_of_Meta.svg.png 2x" title="Bandera del Meta" width="20"/> </span><a href="/wiki/Meta_(Colombia)" title="Meta (Colombia)">Meta</a>
</td>
<td>827 390
</td>
<td>261 359
</td>
<td bgcolor="#00cc33">1 088 749
</td></tr>
<tr align="center">
<td>18
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Caldas" class="thumbborder" data-file-height="700" data-file-width="1000" decoding="async" height="14" src="//upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Flag_of_Caldas.svg/20px-Flag_of_Caldas.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Flag_of_Caldas.svg/30px-Flag_of_Caldas.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Flag_of_Caldas.svg/40px-Flag_of_Caldas.svg.png 2x" title="Bandera de Caldas" width="20"/> </span><a href="/wiki/Caldas" title="Caldas">Caldas</a>
</td>
<td>804 718
</td>
<td>241 700
</td>
<td bgcolor="#00cc33">1 046 418
</td></tr>
<tr align="center">
<td>19
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de La Guajira" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/3/35/Flag_of_La_Guajira.svg/20px-Flag_of_La_Guajira.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/3/35/Flag_of_La_Guajira.svg/30px-Flag_of_La_Guajira.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/3/35/Flag_of_La_Guajira.svg/40px-Flag_of_La_Guajira.svg.png 2x" title="Bandera de La Guajira" width="20"/> </span><a href="/wiki/La_Guajira" title="La Guajira">La Guajira</a>
</td>
<td>507 830
</td>
<td>508 079
</td>
<td bgcolor="#00cc33">1 015 909
</td></tr>
<tr align="center">
<td>20
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Risaralda" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/0/02/Flag_of_Risaralda.svg/20px-Flag_of_Risaralda.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/0/02/Flag_of_Risaralda.svg/30px-Flag_of_Risaralda.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/02/Flag_of_Risaralda.svg/40px-Flag_of_Risaralda.svg.png 2x" title="Bandera de Risaralda" width="20"/> </span><a href="/wiki/Risaralda" title="Risaralda">Risaralda</a>
</td>
<td>792 861
</td>
<td>195 230
</td>
<td bgcolor="#00cc33">988 091
</td></tr>
<tr align="center">
<td>21
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Sucre" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/3/39/Flag_of_Sucre_%28Colombia%29.svg/20px-Flag_of_Sucre_%28Colombia%29.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/3/39/Flag_of_Sucre_%28Colombia%29.svg/30px-Flag_of_Sucre_%28Colombia%29.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/3/39/Flag_of_Sucre_%28Colombia%29.svg/40px-Flag_of_Sucre_%28Colombia%29.svg.png 2x" title="Bandera de Sucre" width="20"/> </span><a href="/wiki/Sucre_(Colombia)" title="Sucre (Colombia)">Sucre</a>
</td>
<td>602 402
</td>
<td>379 325
</td>
<td bgcolor="#00cc33">981 727
</td></tr>
<tr align="center">
<td>22
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Quindío" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/29/Flag_of_Quind%C3%ADo.svg/20px-Flag_of_Quind%C3%ADo.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/29/Flag_of_Quind%C3%ADo.svg/30px-Flag_of_Quind%C3%ADo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/29/Flag_of_Quind%C3%ADo.svg/40px-Flag_of_Quind%C3%ADo.svg.png 2x" title="Bandera de Quindío" width="20"/> </span><a href="/wiki/Quind%C3%ADo" title="Quindío">Quindío</a>
</td>
<td>508 691
</td>
<td>68 852
</td>
<td bgcolor="#00cc33">577 543
</td></tr>
<tr align="center">
<td>23
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Chocó" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Flag_of_Choc%C3%B3.svg/20px-Flag_of_Choc%C3%B3.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Flag_of_Choc%C3%B3.svg/30px-Flag_of_Choc%C3%B3.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Flag_of_Choc%C3%B3.svg/40px-Flag_of_Choc%C3%B3.svg.png 2x" title="Bandera de Chocó" width="20"/> </span><a href="/wiki/Choc%C3%B3" title="Chocó">Chocó</a>
</td>
<td>241 698
</td>
<td>315 956
</td>
<td bgcolor="#00cc33">557 654
</td></tr>
<tr align="center">
<td>24
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Casanare" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Flag_of_Casanare.svg/20px-Flag_of_Casanare.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Flag_of_Casanare.svg/30px-Flag_of_Casanare.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Flag_of_Casanare.svg/40px-Flag_of_Casanare.svg.png 2x" title="Bandera de Casanare" width="20"/> </span><a href="/wiki/Casanare" title="Casanare">Casanare</a>
</td>
<td>321 335
</td>
<td>123 267
</td>
<td bgcolor="#00cc33">444 602
</td></tr>
<tr align="center">
<td>25
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Caquetá" class="thumbborder" data-file-height="504" data-file-width="756" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/b/b0/Flag_of_Caquet%C3%A1.svg/20px-Flag_of_Caquet%C3%A1.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/b/b0/Flag_of_Caquet%C3%A1.svg/30px-Flag_of_Caquet%C3%A1.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/b0/Flag_of_Caquet%C3%A1.svg/40px-Flag_of_Caquet%C3%A1.svg.png 2x" title="Bandera de Caquetá" width="20"/> </span><a href="/wiki/Caquet%C3%A1" title="Caquetá">Caquetá</a>
</td>
<td>278 678
</td>
<td>145 179
</td>
<td bgcolor="#00cc33">423 857
</td></tr>
<tr align="center">
<td>26
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Putumayo" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/93/Flag_of_Putumayo.svg/20px-Flag_of_Putumayo.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/93/Flag_of_Putumayo.svg/30px-Flag_of_Putumayo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/93/Flag_of_Putumayo.svg/40px-Flag_of_Putumayo.svg.png 2x" title="Bandera de Putumayo" width="20"/> </span><a href="/wiki/Putumayo_(Colombia)" title="Putumayo (Colombia)">Putumayo</a>
</td>
<td>193 146
</td>
<td>180 896
</td>
<td bgcolor="#00cc33">374 042
</td></tr>
<tr align="center">
<td>27
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Arauca" class="thumbborder" data-file-height="600" data-file-width="800" decoding="async" height="14" src="//upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Flag_of_Arauca.svg/18px-Flag_of_Arauca.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Flag_of_Arauca.svg/27px-Flag_of_Arauca.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Flag_of_Arauca.svg/36px-Flag_of_Arauca.svg.png 2x" title="Bandera de Arauca" width="18"/> </span><a href="/wiki/Arauca_(Colombia)" title="Arauca (Colombia)">Arauca</a>
</td>
<td>200 433
</td>
<td>107 868
</td>
<td bgcolor="#00cc33">308 301
</td></tr>
<tr align="center">
<td>28
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Vichada" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Flag_of_Vichada.svg/20px-Flag_of_Vichada.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Flag_of_Vichada.svg/30px-Flag_of_Vichada.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Flag_of_Vichada.svg/40px-Flag_of_Vichada.svg.png 2x" title="Bandera de Vichada" width="20"/> </span><a href="/wiki/Vichada" title="Vichada">Vichada</a>
</td>
<td>28 071
</td>
<td>88 873
</td>
<td bgcolor="#00cc33">116 944
</td></tr>
<tr align="center">
<td>29
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Guaviare" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Flag_of_Guaviare.svg/20px-Flag_of_Guaviare.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Flag_of_Guaviare.svg/30px-Flag_of_Guaviare.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/9b/Flag_of_Guaviare.svg/40px-Flag_of_Guaviare.svg.png 2x" title="Bandera de Guaviare" width="20"/> </span><a href="/wiki/Guaviare" title="Guaviare">Guaviare</a>
</td>
<td>52 359
</td>
<td>39 922
</td>
<td bgcolor="#00cc33">92 281
</td></tr>
<tr align="center">
<td>30
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Amazonas (Colombia)" class="thumbborder" data-file-height="500" data-file-width="808" decoding="async" height="12" src="//upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Flag_of_Amazonas_%28Colombia%29.svg/20px-Flag_of_Amazonas_%28Colombia%29.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Flag_of_Amazonas_%28Colombia%29.svg/30px-Flag_of_Amazonas_%28Colombia%29.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Flag_of_Amazonas_%28Colombia%29.svg/40px-Flag_of_Amazonas_%28Colombia%29.svg.png 2x" title="Bandera de Amazonas (Colombia)" width="20"/> </span><a href="/wiki/Amazonas_(Colombia)" title="Amazonas (Colombia)">Amazonas</a>
</td>
<td>42 711
</td>
<td>41 097
</td>
<td bgcolor="#00cc33">83 808
</td></tr>
<tr align="center">
<td>31
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de San Andrés y Providencia" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/3/36/Flag_of_San_Andr%C3%A9s_y_Providencia.svg/20px-Flag_of_San_Andr%C3%A9s_y_Providencia.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/3/36/Flag_of_San_Andr%C3%A9s_y_Providencia.svg/30px-Flag_of_San_Andr%C3%A9s_y_Providencia.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/3/36/Flag_of_San_Andr%C3%A9s_y_Providencia.svg/40px-Flag_of_San_Andr%C3%A9s_y_Providencia.svg.png 2x" title="Bandera de San Andrés y Providencia" width="20"/> </span><a href="/wiki/Archipi%C3%A9lago_de_San_Andr%C3%A9s,_Providencia_y_Santa_Catalina" title="Archipiélago de San Andrés, Providencia y Santa Catalina">San Andrés y Providencia</a>
</td>
<td>46 807
</td>
<td>18 856
</td>
<td bgcolor="#00cc33">65 663
</td></tr>
<tr align="center">
<td>32
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Guainía" class="thumbborder" data-file-height="480" data-file-width="720" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Flag_of_Guain%C3%ADa.svg/20px-Flag_of_Guain%C3%ADa.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Flag_of_Guain%C3%ADa.svg/30px-Flag_of_Guain%C3%ADa.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Flag_of_Guain%C3%ADa.svg/40px-Flag_of_Guain%C3%ADa.svg.png 2x" title="Bandera de Guainía" width="20"/> </span><a href="/wiki/Guain%C3%ADa" title="Guainía">Guainía</a>
</td>
<td>24 958
</td>
<td>27 669
</td>
<td bgcolor="#00cc33">52 627
</td></tr>
<tr align="center">
<td>33
</td>
<td align="left"><span class="flagicon"><img alt="Bandera de Vaupés" class="thumbborder" data-file-height="500" data-file-width="750" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/c/c0/Flag_of_Vaup%C3%A9s.svg/20px-Flag_of_Vaup%C3%A9s.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/c/c0/Flag_of_Vaup%C3%A9s.svg/30px-Flag_of_Vaup%C3%A9s.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/c/c0/Flag_of_Vaup%C3%A9s.svg/40px-Flag_of_Vaup%C3%A9s.svg.png 2x" title="Bandera de Vaupés" width="20"/> </span><a href="/wiki/Vaup%C3%A9s" title="Vaupés">Vaupés</a>
</td>
<td>14 338
</td>
<td>36 795
</td>
<td bgcolor="#00cc33">51 133
</td></tr>
<tr align="center">
<td>
</td>
<th align="left"><b><span class="flagicon"><span style="display:none;">Colombia</span><img alt="Bandera de Colombia" class="thumbborder" data-file-height="600" data-file-width="900" decoding="async" height="13" src="//upload.wikimedia.org/wikipedia/commons/thumb/2/21/Flag_of_Colombia.svg/20px-Flag_of_Colombia.svg.png" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/2/21/Flag_of_Colombia.svg/30px-Flag_of_Colombia.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/21/Flag_of_Colombia.svg/40px-Flag_of_Colombia.svg.png 2x" title="Bandera de Colombia" width="20"/> </span><a href="/wiki/Colombia" title="Colombia">Colombia</a></b>
</th>
<th><b>40 178 069</b>
</th>
<th><b>11 978 245</b>
</th>
<th bgcolor="#00cc33"><b>52 156 314</b>
</th></tr>
</tbody></table>